dspace-server-webapp/src/main/java/org/dspace/app/rest/security/EPersonRestAuthenticationProvider.java

```diff
@@ -122,6 +135,11 @@ private Authentication authenticateNewLogin(Authentication authentication) {
                                      .getHeader(newContext, "login", "type=explicit"));
 
                         output = createAuthentication(newContext);
+
+                        for (PostLoggedInAction action : postLoggedInActions) {
+                            action.loggedIn(newContext);
+                        }
```

Comment:
In my testing, I ran into a severe bug regarding the `PostLoggedInAction`.  If any of these calls to `loggedIn()` throw an error, it blocks all authentication into the system. 

In my situation, I had forgotten to import the new "dspace.object.owner" field.  But that meant that no one could authenticate into the system.

Here's the stacktrace I saw in the logs:
```
ERROR unknown unknown org.springframework.boot.web.servlet.support.ErrorPageFilter @ Forwarding to error page from
request [/api/authn/login] due to exception [No such metadata field: schema=dspace, element=object, qualifier=owner]
java.lang.IllegalArgumentException: No such metadata field: schema=dspace, element=object, qualifier=owner
        at org.dspace.content.ItemServiceImpl.findByAuthorityValue(ItemServiceImpl.java:1315) ~[dspace-api-7.3-SNAPSHOT.jar:7.3-SNAPSHOT]
        at org.dspace.app.profile.ResearcherProfileServiceImpl.findResearcherProfileItemById(ResearcherProfileServiceImpl.java:223) ~[dspac
e-api-7.3-SNAPSHOT.jar:7.3-SNAPSHOT]
        at org.dspace.app.profile.ResearcherProfileServiceImpl.findById(ResearcherProfileServiceImpl.java:93) ~[dspace-api-7.3-SNAPSHOT.jar
:7.3-SNAPSHOT]
        at org.dspace.app.rest.login.impl.ResearcherProfileAutomaticClaim.currentUserHasAlreadyResearcherProfile(ResearcherProfileAutomatic
Claim.java:112) ~[classes/:7.3-SNAPSHOT]
        at org.dspace.app.rest.login.impl.ResearcherProfileAutomaticClaim.claimProfile(ResearcherProfileAutomaticClaim.java:99) ~[classes/:
7.3-SNAPSHOT]
        at org.dspace.app.rest.login.impl.ResearcherProfileAutomaticClaim.loggedIn(ResearcherProfileAutomaticClaim.java:87) ~[classes/:7.3-
SNAPSHOT]
        at org.dspace.app.rest.security.EPersonRestAuthenticationProvider.authenticateNewLogin(EPersonRestAuthenticationProvider.java:140)
~[classes/:7.3-SNAPSHOT]
        at org.dspace.app.rest.security.EPersonRestAuthenticationProvider.authenticate(EPersonRestAuthenticationProvider.java:89) ~[classes
/:7.3-SNAPSHOT]
```

Would there be some way to catch & log these errors **without** blocking authentication? Perhaps this needs to be in some sort of try/catch block?